Assets/MRTK/Core/Utilities/Async/Internal/AsyncCoroutineRunner.cs

```diff
@@ -133,5 +152,19 @@ private void Update()
                 next();
             }
         }
+
+        private void OnDisable()
+        {
+            if (instance == this)
+            {
+                isInstanceRunning = false;
+            }
```

Comment:
We should log a a message if the OnDisabled is called on a AsyncRunner that is not the default instance. Currently, OnDisable will just run silently and not raise any indicators that no action was taken

Reply:
Could you explain why would that be necessary? Disabling an "extra" instance of the runner seems the right thing to me and thus I don't think we should warn user about that. Also we do not disable runner in our code so that is very likely to be the user's action.

Reply:
Didn't realize that this was on top of the existing Monobehavior behavior. Looks good to me